app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -83,10 +82,9 @@ const Summary = ({
           />
         )}
 
-      {status.live && !status.review && !status.endRequested && (
+      {status.live && !status.review && !status.endRequested && status.idle && (
```

Comment:
I wonder if the review and endRequested checks are redundant now...  Don't change it, just curious and thinking out loud.